Assets/MixedRealityToolkit/_Core/Resources/Shaders/3DTextShader.shader

```diff
@@ -0,0 +1,114 @@
+﻿// Upgrade NOTE: upgraded instancing buffer 'Props' to new syntax.
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+///
+/// Basic 3D Text shader with proper z-sorting
+///
+Shader "Mixed Reality Toolkit/3DTextShader"
```

Comment:
I'm not sure we want to bring this over.

Reply:
Wait, I think I added the FontShader3D under Features/UX which has the culling option. Did I add that one too? We should only have one, just let me know where to put it.

Reply:
the shaders need to be availabe to all SDK packages. please co-locate the chosen font shader with the mrtk standard shader. we will ensure (post this pr series) that they are in the correct place. thx